test/__snapshots__/graphql-client.test.ts.snap

```diff
@@ -114,16 +114,7 @@ exports[`graphql-client write batch upsert resetData 2`] = `
 }
 `;
 
-exports[`graphql-client write batch upsert resetData 3`] = `
-"mutation {
-      
-      del: delete_vcs_Organization(where:{id: {_in:[
-        "00042ddb78c1a5956320e7fbc7ca488517a9caa8","0008d73ec6b79910fcbadb7410c4d191e92ce44d","0011f5f8600d14dae163a37286aa1cd5e6bf6a71"
-      ]}}) {
-        affected_rows
-      }
-    }"
-`;
+exports[`graphql-client write batch upsert resetData 3`] = `"mutation { del: delete_vcs_Organization (where: {_and: {refreshedAt: {_lt: "2200-01-01T00:00:00.000Z"}, origin: {_eq: "foo"}, id: {_in: ["00042ddb78c1a5956320e7fbc7ca488517a9caa8", "0008d73ec6b79910fcbadb7410c4d191e92ce44d", "0011f5f8600d14dae163a37286aa1cd5e6bf6a71"]}}}) { affected_rows } }"`;
```

Comment:
Hmm... the formatting looks a little off, I would expect an `[]` after the `_and` but maybe that is alright?

Reply:
check if the query works in GraphiQL

Reply:
_and requires [] field.  I think this is incorrect.